File: PHP/src/Jakopec/PartijaDvaIgraca.php
<?php


namespace Jakopec;


class PartijaDvaIgraca extends Partija
{
    public function __toString(): string
    {

        $rezultat = $this->getRezultat();
   return 'Partija DVA IGRAČA, igra gotova: ' . $this->isIgraGotova() . ', ' . $this->getIgraci()[0] . ' i ' . ': ' .
       $rezultat->getPrvi() .
       ' | ' . $this->getIgraci()[1] . ': ' . $rezultat->getDrugi();
    }
}

File: Java/src/jakopec/model/PartijaDvaIgraca.java
package jakopec.model;

public class PartijaDvaIgraca extends Partija {

    @Override
    public String toString() {
        Rezulat rezultat = getRezultat();
        return "Partija DVA IGRAČA, igra gotova: " + isIgraGotova() + ", " + getIgraci().get(0) + ": " +
                rezultat.getPrvi() +
                " | "+ getIgraci().get(1) + ": " + rezultat.getDrugi();
    }

}
